libavcodec/qsvenc.c

```diff
@@ -146,6 +146,15 @@ static const struct {
     { MFX_RATECONTROL_QVBR,    "QVBR" },
 };
 
+#define UPDATE_PARAM(a, b)  \
+do {                        \
+    if ((a) != (b))         \
+    {                       \
```

Comment:
`if` and `{` are the same line in FFmpeg.

Reply:
Thanks
Done.